### DISCUSSION TITLE
build instruction?

### DISCUSSION THREAD

**Discussion by tulx17** [massCodeIO/massCode#139] [Q&A]
I tried to build an AppImage but some missing file make the build failed.
![image](https://user-images.githubusercontent.com/50549524/179403185-b6540fc5-04fe-4e79-8eef-b585849605c6.png)

**Comment by antonreshetov** [comment]
https://github.com/massCodeIO/massCode/discussions/87

**Reply by tulx17** [reply]
I did that already but the problem still persists

**Reply by antonreshetov** [reply]
Apparently there is still a style import that is not being used. Simply delete the import - https://github.com/massCodeIO/massCode/blob/master/src/renderer/assets/scss/vendor.scss#L1

**Reply by tulx17** [reply]
And now it can't run. Guess I have no luck with this then.
![image](https://user-images.githubusercontent.com/50549524/180722036-c02895b4-6b66-4674-bad7-ff5d18377a18.png)

**Comment by antonreshetov** [comment]
Perhaps you should consult Linux users
